src/core_atmosphere/dynamics/mpas_atm_time_integration.F

```diff
@@ -271,6 +276,22 @@ subroutine mpas_atm_dynamics_init(domain)
 
       call mpas_pool_get_array(mesh, 'bdyMaskEdge', bdyMaskEdge)
       !$acc enter data copyin(bdyMaskEdge)
+
+      call mpas_pool_get_array(mesh, 'zz', zz)
+      !$acc enter data copyin(zz)
```

Comment:
Collecting the `copyins` would likely be helpful. I think the way it's written now could help with debugging whether a `pool_get_array` or a `copyin` fails.

We really should look at adding the timer! Probably some `..._init` version so it can be separate from the time spent during timesteps.

Reply:
As @gdicker1 alluded to, this code in `mpas_atm_dynamics_init` is only run once when the model starts up, and so we haven't bothered to add timers for it. We're currently using timers in the computational routines that are invoked every timestep to get a sense for how much faster (hopefully not slower!) those routines are running on GPUs, and for that it's important to be able to subtract out the data movement overhead in every timestep that will eventually be eliminated. But adding a timer to the initial data movement could definitely be the purpose of a future PR.

Reply:
@gdicker1 @mgduda 
Thanks for the clarifications.